docs/getting-started.md

```diff
@@ -107,21 +114,27 @@ You can test everything is working by running `ngrok -h` which should print the
     OPTIONS:
       -h, --help      help for ngrok
       -v, --version   version for ngrok
+  ```
 
 ### Step 3: Connect your agent to your ngrok account
 
 Now that the ngrok agent is installed, let's connect it to your ngrok Account. If you haven't already, [sign up (or log in)](https://dashboard.ngrok.com) to the ngrok Dashboard and get your [Authtoken](https://dashboard.ngrok.com/get-started/your-authtoken). The ngrok agent uses the authtoken (sometimes called tunnel credential) to log into your account when you start a tunnel.
 
 Copy the value and run this command to add the authtoken in your terminal.
 
-    ngrok config add-authtoken TOKEN
+```bash
+ngrok config add-authtoken TOKEN
+``` 
 
 ### Step 4: Start ngrok
 
 Start ngrok by running the following command.
 
-    ngrok http 8000
+```bash
+ngrok http 8000
+```
 
+```
 You should see something similar to the following console UI in your terminal.
 
```

Comment:
@avb-is-me . The code block should start after "You should see something similar to the following console UI in your terminal."

Reply:
@sudobinbash done!